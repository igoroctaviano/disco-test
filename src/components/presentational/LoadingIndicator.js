import React from "react";

/* Assets */
import loadingIcon from "../../assets/img/loader.gif";

const LoadingIndicator = () => (
  <div className="loading-indicator">
    <img src={loadingIcon} alt="Loading Indicator" />
  </div>
);

export default LoadingIndicator;
