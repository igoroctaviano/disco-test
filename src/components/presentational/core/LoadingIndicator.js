import React from "react";
import loadingIcon from "../../../assets/images/loader.gif";

const LoadingIndicator = () => (
  <div className="loading-indicator">
    <img src={loadingIcon} alt="Loading Indicator" />
  </div>
);

export default LoadingIndicator;
