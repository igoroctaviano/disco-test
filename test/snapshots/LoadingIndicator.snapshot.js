import React from "react";
import LoadingIndicator from "./../../src/components/presentational/core/LoadingIndicator.js";
import renderer from "react-test-renderer";

test("renders correctly", () => {
  const tree = renderer.create(<LoadingIndicator />).toJSON();
  expect(tree).toMatchSnapshot();
});
