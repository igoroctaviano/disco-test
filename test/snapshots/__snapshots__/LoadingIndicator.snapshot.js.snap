// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="loading-indicator"
>
  <img
    alt="Loading Indicator"
    src={
      Object {
        "process": [Function],
      }
    }
  />
</div>
`;
